frontend/app/mixins/model-truncated-details.js

```diff
@@ -1,23 +1,25 @@
 import Ember from 'ember';
 
 export default Ember.Mixin.create({
-    truncatedBio: function(key, value) {
-      // setter
-      if (arguments.length > 1) {
+    truncatedBio: Ember.computed('bio', {
```

Comment:
Is this new syntax only used for computed setters, or are we just taking it slow transitioning away from `.property()`?